plugins/woocommerce/tests/e2e-pw/tests/admin-analytics/analytics-overview.spec.js

```diff
@@ -72,19 +72,23 @@ test.describe( 'Analytics pages', () => {
 			await page.goto(
 				'wp-admin/admin.php?page=wc-admin&path=%2Fanalytics%2Foverview'
 			);
-			// check the top section (Performance)
-			await page.click(
-				'//button[@title="Choose which analytics to display and the section name"]'
-			);
+			// check the top section
+			await page
+				.locator( 'button.woocommerce-ellipsis-menu__toggle' )
+				.first()
+				.click();
```

Comment:
In localhost, the site always starts with "Performance" as the first section. But that's not the case in permanent sites like our daily smoke test site. Sometimes, the test would start with a different section at the top. To address this, I "generalized" this step to click on the first ellipsis menu, regardless of what section it is.